### ISSUE TITLE
Core Helicopters is hurting the mods abillity to flex Race's (Everyone)  strength and Diversity.

### ISSUE THREAD

**Issue by Thorgram** [techannihilation/TA_O#1321]
Core Helipad Tier is a dead-end tech that limits their ability to abuse the originally intended strategy of being bullies. Its so bad that in fact not only hurts Cores Theme, build-paths, but severely hampers Arm and the Lost Legacy ability to be ethically diverse. The immediate removal and regrowth will help everyone. I also have the understanding that races are too similar in certain aspects but we don't want to simply gut as that it hurts diversity inside the current race we really want to fix (core).

The first thing we should address is the direct effect of the balancing decisions within the Core race itself.
I believe that the developing team feels a deep desire to try and hide severe weaknesses and builds counters that should not exist in the first place. When Arm Spiders as a lab was implemented there was a fear that Arm dominates hillly maps and Core has no space to counterplay. This is wrong as Core players should be more grounded in establishing base before pushing back with their stronger late game armored units. Of course Core had two course of action, fast air response to punish spider attacks or build a defense wall.

So what does this have anything to do with Core Helicopters? Well, at the time aircraft and helicopters were really the only other all-terrain counter attackers possible. Now some would argue why was helicopters implemented in the first place, but that is another discussion. I actually like the concept, however Core does not seem like the type of Race that would spin a blade to manipulate the existing flow. 

Rather as dictated by their https://cdn.discordapp.com/attachments/431389653174845441/472085151027822602/unknown.png ship on bottom left, anti gravity pod wings and thrusters makes bit more sense. In fact, instead of helicopters, they would be most likely be using small low surface skycruisers or spaceships to try and dominate the airspace instead of helicopters. As different, these low slow anti gravity ships would have a huge hull advantage vs. airplanes and sit rather than strafe like most gunship class vehicles. This actually gives Core the all-terrain capabilities it needs to counter attack in spider routes. Nothing is scary then slow floating swarms of iron rolling down the hill and descending on upon the villages of the poorly defended. The Helipad can still be used or replaced with another kind of shipyard that accommodates the shape of new aerial vessels.

However when the counter to all-terrain spiders were decided, other unique facilities filled the role to accommodate the issues that Core could not handle the all-terrain issue. Its very possible that gunship helicopters if they weren't so sub-par for cost vs spiders could have saved the modding community time from having to create a new lab for the lost legacy and core race. Because all the races now have all-terrain it greatly hurts the feel and the strength and weaknesses of each race. This is a problem that does not exist, should have not been solved.

TL:DR Forget Helicopters, as the President of the Core, I decided we needed a new branch in the military called the Space Force. Also Helicopters are so 1950s.

**Comment by skymyj** [comment]
Nixtux will think, I spoke you about copter. But even not.
I'm agree with your mind. And ATM, we have the modeling capacity to create new ship using anti gravity pod wings and thrusters.

**Comment by nixtux** [comment]
Removing copter no, but a space age replacement is not an issue

**Comment by Leggir** [comment]
Umm, Core has Tortoise units which easily counter spiders.
